util/check_characters.py

```diff
@@ -37,7 +40,14 @@ def main():
             try:
                 next(reader, None)  # skip the file header (i.e. "transcript")
                 for row in reader:
-                    all_text |= set(str(row[2]))
+                    if args.unicode_variants:
+                        unicode_transcript = unicodedata.normalize("NFKC", row[2])
+                        if row[2] != unicode_transcript:
+                            raise ValueError(row[2])
+                    all_text |= set(row[2])
+            except ValueError as e:
```

Comment:
Why are you raising and catching in the same scope? Just print the message and exit directly in the if on line 46?

Reply:
That was just to make the flow nicer